src/style-stars--1.css

```diff
@@ -0,0 +1,3 @@
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../stars/yellow_star.png);
+}
```

Comment:
It seems like you are directly styling an element using a tag name in combination with a class. According to the checklist, you should avoid using tag names for styling, except for `html` and `body`. Instead, apply styles using classes to ensure that your CSS is more modular and avoids unintended side effects. Consider adding a class to the `stars__star` elements and style them using that class.